File: proje-C#/KampIntro/GameProject/CampaignManager.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GameProject
{
    class CampaignManager : ICampaignService
    {
        public static int DiscountRate { get; internal set; }
        public string CampanignName { get; private set; }

        public void Add(CampaignManager campaign)
        {
            Console.WriteLine( campaign .CampanignName  +  "  "  + "isimli kampanya eklendi");
        }

        public void Delete(CampaignManager campaign)
        {
            Console.WriteLine(campaign.CampanignName + "  " + "isimli kampanya silindi");
        }

        public void Update(CampaignManager campaign)
        {
            Console.WriteLine(campaign.CampanignName + "  " + "isimli kampanya güncellendi");
        }

        internal void Add(Campaign campaign)
        {
            throw new NotImplementedException();
        }
    }
}

File: proje-java/GameStore/src/Concrete/CampaignManager.java
package Concrete;

import Abstract.CampaignService;
import Entities.Campaign;

public class CampaignManager implements CampaignService {

	
	


	@Override
	public void Add(Campaign campaign) {
		System.out.println(campaign + "Kayýt eklendi");
		
	}

	@Override
	public void Update(Campaign campaign) {
		System.out.println(campaign + "Kayýt güncellendi");
		
	}

	@Override
	public void Delete(Campaign campaign) {
		System.out.println(campaign + "Kayýt silindi");		
	}
	
}	 

	